rotkehlchen/tasks/manager.py

```diff
@@ -500,6 +506,31 @@ def _maybe_update_snapshot_balances(self) -> None:
                     ignore_cache=True,
                 )
 
+    def _maybe_update_curve_pools(self) -> None:
+        curve_caches = GlobalDBHandler().get_cache_keys(
+            cache_type=GeneralCacheType.CURVE_POOL_ADDR,
+        )
+        if len(curve_caches) == 0:
+            schedule = True
```

Comment:
`should_schedule` sounds better, yeah. But I am not sure about moving this variable out. I put it in to be sure that we don't forget to handle any case here (since `variable might be referenced before assignment` warning will be shown otherwise)